runtime/src/iree/hal/drivers/cuda/dynamic_symbols.c

```diff
@@ -163,11 +196,39 @@ iree_status_t iree_hal_cuda_nccl_dynamic_symbols_initialize(
   return status;
 }
 
+// FIXME(okkwon): it is unrelated to CUDA, but iree_hal_cuda_dynamic_symbols_t
```

Comment:
yeah, we'll need to move this out to iree/hal/utils/ or something in the future and decouple it so cpu/etc can use it, but this is fine to start